README.md

```diff
@@ -92,42 +97,40 @@ If it can be trimmer, smoke-and-mirrors likes to trim it.
 [![devDependency Status](https://david-dm.org/html-next/vertical-collection/dev-status.svg)](https://david-dm.org/html-next/vertical-collection#info=devDependencies)
 [![Coverage Status](https://coveralls.io/repos/html-next/vertical-collection/badge.svg?branch=master&service=github)](https://coveralls.io/github/html-next/vertical-collection?branch=master)
 
-
 ## Documentation
 
 For updated documentation and demos see [http://html-next.github.io/vertical-collection/](http://html-next.github.io/vertical-collection/)
 
 ## Contributing
 
- - Open an Issue for discussion first if you're unsure a feature/fix is wanted.
- - Branch off of `master` (default branch)
- - Use descriptive branch names (e.g. `<type>/<short-description>`)
- - Use [Angular Style Commits](https://github.com/angular/angular.js/blob/v1.4.8/CONTRIBUTING.md#commit)
- - PR against `master` (default branch).
+- Open an Issue for discussion first if you're unsure a feature/fix is wanted.
+- Branch off of `master` (default branch)
+- Use descriptive branch names (e.g. `<type>/<short-description>`)
+- Use [Angular Style Commits](https://github.com/angular/angular.js/blob/v1.4.8/CONTRIBUTING.md#commit)
+- PR against `master` (default branch).
 
 ### Commits
```

Comment:
@pzuraq what would you think of removing this whole section on commits? I think using labels in PRs works best for changelog generation these days.